### ISSUE TITLE
Custom path to nanoc.yaml

### ISSUE THREAD

**Issue by axilleas** [nanoc/features#22]
There are special cases where you need to specify a different `nanoc.yaml` than that in the root dir.

It would be nice if you could do that :)

**Comment by denisdefreyne** [comment]
Definitely useful!

**Comment by denisdefreyne** [comment]
Are you thinking something along the lines of

```
nanoc -f ../nanoc_whatever.yaml
```

?

It might be possible to do what you want to do with environments (see the [environments documentation](https://nanoc.ws/doc/sites/#environments)).

**Comment by axilleas** [comment]
Yep, something like that! And the environments is indeed a good idea to overcome my issue :) Thanks!
